File: src-gen/Cinema/Cinema/ClientApp/src/api/models/bookings/CinemaBooking.ts
import {Client} from "../customers/Client"
import {Seat} from "../resources/Seat"
import {NightPlan} from "../schedules/NightPlan"

export type CinemaBooking = {
	id: string
	name: string
	client: Client
	seat: Seat
	plan: NightPlan
} 

File: src-gen/Cinema/Cinema/Persistence/Models/CinemaBooking.cs
using System;
using System.Collections.Generic;

namespace Cinema.Persistence.Models
{
    public class CinemaBooking : IEntity
    {
    	public Guid Id {get; set;}
        public string name {get; set;}
        public Client client {get; set;} 
        public Seat seat {get; set;} 
        public NightPlan plan {get; set;} 
    }
}
